test/e2e/e2etest.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+# Keep track of the script directory
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+# OpenSDS Root directory
+OPENSDS_DIR=$(cd $TOP_DIR/../.. && pwd)
+
+split_line(){
+    echo "================================================================================================"
+    echo $*
+    echo "================================================================================================"
+}
+
+# Start lvm e2e test
+split_line "Start lvm e2e test"
```

Comment:
only lvm e2e test?

Reply:
This will trigger the test files in the path: opensds/test/e2e/e2e_test.go
its having volume related e2e testcases, each respective features e2e cases can be kept in this path.. so that, CI will trigger those cases